zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -133,20 +134,23 @@
       "editOnDblClick", false);
 
   public InterpreterFactory(ZeppelinConfiguration conf,
-      AngularObjectRegistryListener angularObjectRegistryListener,
-      RemoteInterpreterProcessListener remoteInterpreterProcessListener,
-      ApplicationEventListener appEventListener, DependencyResolver depResolver,
-      boolean shiroEnabled) throws InterpreterException, IOException, RepositoryException {
+                            AngularObjectRegistryListener angularObjectRegistryListener,
+                            RemoteInterpreterProcessListener remoteInterpreterProcessListener,
```

Comment:
Could you please make sure that all the code changes adhere Zeppelin's [project style guide](https://zeppelin.apache.org/contribution/contributions.html#code-convention)?

In particular, [horizontal code alignment is discouraged](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment) by project's Java language style guide due to mainanability concerns. 

Could you please change it back here and everywhere below?

Ther checkstyle right now verifies only sub-set of styleguide preferences and there is on-going (but stalled) effort to improve it, introducing appropriate consistent cross-IDE formating in #1168